netty/src/main/java/io/grpc/netty/NettyClientHandler.java

```diff
@@ -530,6 +538,10 @@ public void operationComplete(ChannelFuture future) throws Exception {
                     stream.getStatsTraceContext().clientOutboundHeaders();
                     http2Stream.setProperty(streamKey, stream);
 
+                    if (GrpcUtil.shouldBeCountedForInUse(command.callOptions())) {
+                      inUseState.updateObjectInUse(http2Stream, true);
```

Comment:
Let's put a comment here that this delays the inUse state until the I/O completes, which technically may be later than we would like.